.github/workflows/prepare-changelog.yml

```diff
@@ -1,8 +1,20 @@
 name: 'Prepare Changelog'
 on:
+  # Use labels on PRs to update the changelog
+  # - patch
+  # - minor
+  # - major
+  pull_request:
+    types:
+      - labeled
   push:
     branches:
       - main
+
+concurrency:
+  group: ci-${{ github.head_ref || github.ref }}
```

Comment:
I don't think you meant this to share the same concurrency group with the test suite?

Reply:
correct, thanks for catching that -- updated